org-brain.el

```diff
@@ -282,7 +326,8 @@ Ignores \"dotfiles\"."
              (when (and id (not (org-brain-entry-at-point-excludedp)))
                (push (list
                       (org-brain-path-entry-name file)
-                      (org-entry-get (point) "ITEM")
+					  (org-brain-replace-links-with-visible-parts
```

Comment:
It seems like `org-brain-replace-links-with-visible-parts` isn't needed here, since it is run in `org-brain-headline-at` anyway, if needed.